.php_cs

```diff
@@ -0,0 +1,38 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(__DIR__)
+;
+
+return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::NONE_LEVEL)
```

Comment:
you could use constant `Symfony\CS\FixerInterface::SYMFONY_LEVEL` and only add your custom rules.